tests/symlinked_included_files.test.ts

```diff
@@ -60,8 +61,9 @@ test.skipIf(platform() === 'win32')('Symlinked directories from `includedFiles`
 
   // expect that the symlink for `node_modules/crazy-dep` is preserved
   expect(await readDirWithType(unzippedPath)).toEqual({
-    '___netlify-bootstrap.mjs': false,
+    [BOOTSTRAP_FILE_NAME]: false,
```

Comment:
let's keep this as a string. we want a change in the constant to fail some tests, because it's part of the contract we have with e.g. functions origin